articles/connections/enterprise/azure-active-directory-classic.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Connect Azure Active Directory with Auth0 (Classic Portal)
+description: How to obtain a ClientId and Client Secret for a Microsoft Azure Active Directory with the Classic Portal.
+---
+
+# Obtain a *ClientId* and *Client Secret* for a Microsoft Azure Active Directory- Classic Portal
+
+::: panel-info Notice
+This page uses the Azure Active Directory Classic Portal, for information on using the current portal, [click here.](/connections/enterprise/azure-active-directory)
+:::
+
+To allow users to login using a Microsoft Azure Active Directory account, you must register your application through the Microsoft Azure portal. If you don't have a Microsoft Azure account, you can [signup](http://www.windowsazure.com/en-us/pricing/free-trial) for free.
+
+**NOTE:** There is no way to create an application that integrates with Microsoft Azure AD without having **your own** Microsoft Azure AD instance.
+
+## 1. Create a new Microsoft Azure Active Directory instance
+
+Login to Microsoft Azure and click on **Active Directory** on the Dashboard.
+
+![](/media/articles/connections/enterprise/azure-active-directory/waad-0.png)
+
+Click on **ADD+** at the bottom of the screen.
+
+![](/media/articles/connections/enterprise/azure-active-directory/waad-1.png)
+
+Enter a *subdomain*, e.g.: **${account.tenant}**. This can be any text. (It does not have to match the Auth0 subdomain and it will be used in the next step.) Also enter your country and a friendly name for your organization.
```

Comment:
`This can be any text. (It does not have to match the Auth0 subdomain and it will be used in the next step.) Also`: I think you should remove the parenthesis (not the text, just the brackets).